src/python/grpcio/tests/_runner.py

```diff
@@ -43,6 +43,8 @@
 from tests import _loader
 from tests import _result
 
+_READ_BYTES = 2**20
```

Comment:
How about a comment on this constant justifying its existence by explaining everything we've learned today?